tests/session/session-endSession-001.phpt

```diff
@@ -0,0 +1,77 @@
+--TEST--
+MongoDB\Driver\Session::endSession() Calling methods after session has been ended
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_clean(DATABASE_NAME, COLLECTION_NAME); ?>
```

Comment:
This can be removed, as the test doesn't insert anything.

Also, `skip_if_not_clean()` should never be called before checks for a live server. It looks like this was copy/pasted from the `transaction-integration` tests. I've opened https://github.com/mongodb/mongo-php-driver/pull/886 to clean up various SKIPIFs in the session tests.